packages/webpack/test/e2e-scuttle.spec.js

```diff
@@ -68,22 +76,53 @@ test(`webpack/scuttled - hosting globalThis's "Function" is scuttled`, async (t)
 })
 
 test(`webpack/scuttled - hosting globalThis's "Function" is scuttled excepted`, async (t) => {
-  await scuttle(t, {enabled: true, exceptions: ['Function']})
+  await scuttle(t, { enabled: true, exceptions: ['Function'] })
   t.notThrows(() => {
     t.is(new t.context.globalThis.Function('return 1')(), 1)
   }, 'Unexpected error in scenario')
 })
 
+test(`webpack/scuttled - webpackChunk global is transparently added to exceptions`, async (t) => {
+  await scuttle(t, { enabled: true, exceptions: ['Function'] })
+
+  t.notThrows(() => {
+    t.context.globalThis.webpackChunkTEST
+  }, 'Unexpected error in scenario')
+  t.truthy(t.context.globalThis.webpackChunkTEST)
+})
+
+test(`webpack/scuttled - webpackChunk global is transparently added to exceptions even when not specified 1`, async (t) => {
+  await scuttle(t, { enabled: true })
+
+  t.notThrows(() => {
+    t.context.globalThis.webpackChunkTEST
+  }, 'Unexpected error in scenario')
+  t.truthy(t.context.globalThis.webpackChunkTEST)
+})
+
+test(`webpack/scuttled - webpackChunk global is transparently added to exceptions even when not specified 2`, async (t) => {
+  await scuttle(t, true)
+
+  t.notThrows(() => {
+    t.context.globalThis.webpackChunkTEST
+  }, 'Unexpected error in scenario')
```

Comment:
[suggestion] this is redundant with the next assertion

```suggestion
```

Reply:
It indicates that the configuration not working results in an error from scuttling. I'm not sure I like the raw error being thrown.
And the second assertion is for making sure that the first didn't succeed because there's no webpackChunkTEST field.
I'd consider changing the assertion message, but I'd like to keep the assertions.